### ISSUE TITLE
File name is not displaying after you attach them to the project.

### ISSUE THREAD

**Issue by dlopez079** [dlopez079/LARAJET-Themischimney.com#28]
When you attach a file to the project, the name of the file is not displayed in the attachment section.  

Since AWS S3 Bucket does not accept multiple files at a time, I had to remove the file upload input attribute multiple.  Since I cannot upload multiple files, I may need to remove the files array which is no longer needed.  

The 'for each' loop is looping through an empty array.

**Comment by dlopez079** [comment]
The public files [] array was created so I can save multiple attachments at once.  AWS does not allow you to upload multiple attachments.  I removed the public files [] array and replaced it with a public variable 'file' with no array and in the singular form.  Then removed the 'for each' loop so there is no array to loop through.  I successfully can attach files to the project.

I encountered an issue where I could not save a second attachment.  I resolved this issue by emitting a 'refreshProjectReview'  function on the Project View Component rather than 'showProjectCard' on the Project View Component.  You don't have to pass any parameters on the refresh function as opposed to the show project card function.
